### ISSUE TITLE
Is it possible to prevent popper from flipping and lock it's placement?

### ISSUE THREAD

**Issue by martinszeltins** [valgeirb/vue3-popper#31]
Even though I have set the placement to "top" it still flips to bottom when it want to. Is there a way to force placement to always be top?

````
<Popper :hover="true" arrow placement="top" ...>
    ....
</Popper>
````

**Comment by valgeirb** [comment]
Not currently, it automatically flips if it runs out of space when scrolling etc. But you raise a good point, maybe this should be optional, I'll look into it.

**Comment by valgeirb** [comment]
Published v1.3.0 which includes the `locked` prop #38
